src/services/withdrawService.ts

```diff
@@ -199,6 +199,14 @@ async function determineNEP141StorageRequirement(
     return { tag: "ok", value: null }
   }
 
+  // For withdrawing ETH to NEAR no storage_deposit is required. (P.S. aurora is ETH address on Near network)
+  if (
+    formValues.tokenOut.chainName === "near" &&
+    formValues.tokenOut.address === "aurora"
+  ) {
+    return { tag: "ok", value: null }
+  }
```

Comment:
I think this check should be moved to `getNEP141StorageRequired`, since that is general function for getting storage. This functions handles withdrawal specific logic for storage.